### PR TITLE
Fixed publish and continue

### PR DESCRIPTION
**Pull request by Aca-jov**
SDNTB-301

### CODE REVIEW

File: scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js [hunk 1]
```diff
@@ -394,11 +395,7 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
             }
 
             $scope.useTansaProofing = function () {
-                if ($rootScope.config.features && $rootScope.config.features.useTansaProofing) {
-                    return true;
-                } else {
-                    return false;
-                }
+                return ($rootScope.config.features && $rootScope.config.features.useTansaProofing) ? true : false;
```
Comment (petrjasek) [inline scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js]: imo it's better to write:
```
return $rootScope.config.features && $rootScope.config.features.useTansaProofing;
```
and not adding that extra `? true : false;` part

Comment (Aca-jov) [inline scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js]: Fixed

File: scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js [hunk 2]
```diff
@@ -423,18 +420,19 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
 
             $rootScope.publishAfterTansa = function () {
                 if (!onlyTansaProof) {
-                    $scope.saveTopbar().then($scope.publish);
+                    $scope.saveTopbar().then(continueAfterPublish == true ? $scope.publishAndContinue : $scope.publish);
```
Comment (petrjasek) [inline scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js]: here `== true` is extra

File: scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js [hunk 3]
```diff
@@ -367,7 +368,7 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                     }
                 }
 
-                return false;
+                return target.length == 0 ? true : false;
```
Comment (petrjasek) [inline scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js]: `return target.length === 0;`

File: scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js [hunk 4]
```diff
@@ -471,11 +469,7 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
             };
 
             $scope.showCustomButtons = function(item) {
-                if (!(item.task && item.task.desk) || item.state === 'draft' && !$scope.dirty) {
-                    return false;
-                } else {
-                    return true;
-                }
+                return (!(item.task && item.task.desk) || item.state === 'draft' && !$scope.dirty) ? false : true;
```
Comment (petrjasek) [inline scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js]: `return !(!(item.task && item.task.desk) || item.state === 'draft' && !$scope.dirty);`

Comment (Aca-jov) [inline scripts/superdesk-authoring/authoring/directives/AuthoringDirective.js]: All fixed.

**Pull request merged**